docs/proposals/dedicated-nodes.md

```diff
@@ -0,0 +1,210 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/dedicated-nodes.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Dedicated Nodes, Taints, and Tolerations
+
+## Introduction
+
+This document proposes an approach to implementing *dedicated nodes* in Kubernetes.
+A *dedicated node* is a node that can only be used by a limited set of users.
+Dedicated nodes usually occur in sets, with each set of dedicated nodes being
+associated with a particular set of users. There are many possible generalizations of dedicated
+nodes, for example a set of nodes with a particular piece of hardware could be dedicated to
+pods that require that hardware.
+
+In theory, dedicated nodes can be implemented using
```

Comment:
Two responses:
(1) This feature will not just be used for dedicated nodes; it will also be used for machine problems and corresponding forgiveness. (#1574)
(2) RedHat has already expressed a need for dedicated nodes (or at least equivalent functionality); see #17151